ubuntu/scripts/cpanel/cleanup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+clean_garbage()
+{
+	# Clean package manager caches
+	if [[ -f "/usr/bin/apt-get" ]]; then
+		if ! apt-get clean; then
+			sleep 120
+			apt-get clean
+		fi
+		rm -rf /var/lib/apt/lists/*
+	fi
+
+	if [[ -f "/usr/bin/yum" ]]; then
+		yum clean all
+		rm -rf /var/cache/yum /var/tmp/yum-*
+	fi
+
+	if [[ -f "/usr/bin/dnf" ]]; then
+		dnf clean all
+	fi
+
+	echo "==> Cleaning up leftover dhcp leases"
+	if [[ -d "/var/lib/dhcp" ]]; then
+		rm -rf /var/lib/dhcp
+	fi
+
+  # Remove default network configs
+  rm -f /etc/sysconfig/network-scripts/ifcfg-ens3
+
+	# Clean /tmp
+	rm -rf /tmp/*
+
+	# Clean SSH Host Key Pairs
+	rm -rf /etc/ssh/*_key /etc/ssh/*_key.pub
+
+	if [[ -d /root/.ssh ]]; then
+		# Clean keys for root user
+		rm -f /root/.ssh/authorized_keys
+
+		echo -n > /root/.ssh/known_hosts
+		chmod 0644 /root/.ssh/known_hosts
+	fi
+
+  # Lock root password
+  passwd -l root
+
+	# Clean up log files
+	find /var/log -type f | while read f; do echo -ne '' > ${f}; done;
+
+	# Clean evidences of your activity
+	echo -n > /root/.bash_history
+
+	# Insert Gooogle resolver cause PowerDNS installed by cPanel disables systemd-resolved
```

Comment:
Please add a link to a cPanel's issue.

Reply:
I double checked - cPanel on Ubuntu 20.04 supports only PowerDNS for nameservers:
https://docs.cpanel.net/installation-guide/system-requirements-ubuntu/#nameservers

PowerDNS requires port 53, which is by systemd-resolved is automatically disabled  during cPanel installation and content of /etc/resolved.conf is replaced with 8.8.8.8 and 1.1.1.1.

This forum topic describes the same behavior, only in this case due to specific configuration of the customer DNS resolution on his server was broken
https://forums.cpanel.net/threads/install-issue-dns-changing.712857/